control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml

```diff
@@ -75,8 +75,6 @@ spec:
               value: kafka-channel-channels-subscriptions
             - name: SINK_CONTRACT_CONFIG_MAP_NAME
               value: kafka-sink-sinks
-            - name: SOURCE_CONTRACT_CONFIG_MAP_NAME
-              value: kafka-source-sources
```

Comment:
I think it is there because it's a required configuration in the struct but we don't use it?

Reply:
We don't use the Env struct for Source anywhere so this is not required, I believe